testing/tests/DevExpress.animation/position.tests.js

```diff
@@ -955,6 +955,34 @@ const testCollision = (name, fixtureName, params, expectedHorzDist, expectedVert
         }
     });
 
+    QUnit.test('position should return window.height() if window.outerHeight == window.innerHeight (T939748)', function(assert) {
+        const isPhone = devices.real().deviceType === 'phone';
```

Comment:
Is it actual only for phones? Are the tablets OK?

Reply:
There is the same behavior on tablets as on desktops